web/client/translations/data.de-DE

```diff
@@ -62,8 +62,8 @@
             "featureTypeError": "Layerattribute können nicht geladen werden",
             "featureTypeErrorInvalidJSON": "Layerattribute können nicht geladen werden. Antwort ist nicht gültig",
             "elevation": "Höhe",
-            "titleTranslations": "Title translations",
-            "groupProperties": "Group properties",
+            "titleTranslations": "Titel Übersetzungen",
+            "groupProperties": "Proprietà del gruppo",
```

Comment:
"Proprietà del gruppo" is Italian, it should be  "Gruppeneigenschaften"

Reply:
Yes, please fix this.